dll/win32/shell32/CDefView.cpp

```diff
@@ -160,6 +161,7 @@ class CDefView :
 
     BOOL                      m_isEditing;
     BOOL                      m_isParentFolderSpecial;
+    BYTE                      m_ScheduledStatusbarUpdate;
```

Comment:
```suggestion
    BOOL                      m_ScheduledStatusbarUpdate;
```
(or the C++ `bool` which is 1-byte long and would be closer to your idea of reducing structure size... but since this is 4-byte or 8-byte aligned, in this case this wouldn't matter at all, unless all the other BOOLs in there were converted as well, which is off-topic)